src/gsy_e/gsy_e_core/cli.py

```diff
@@ -106,7 +108,7 @@ def main(log_level):
                 "(orders can't contain attributes/requirements)."))
 def run(setup_module_name, settings_file, duration, slot_length, tick_length,
         cloud_coverage, compare_alt_pricing, enable_external_connection, start_date,
-        pause_at, slot_length_realtime, enable_dof: bool, **kwargs):
+        pause_at, iterative: bool, slot_length_realtime, enable_dof: bool, **kwargs):
```

Comment:
I think we could find a different word for it though. "Iterative" is not exactly the same as "step-by-step" I think, but I might be wrong.

Reply:
I see, I'm also not sure myself but I think that we can find some candidates [here](https://www.merriam-webster.com/thesaurus/step-by-step)